src/Misc/fileTools.ts

```diff
@@ -87,7 +87,11 @@ export class FileTools {
      * Gets or sets a function used to pre-process url before using them to load assets
      */
     public static PreprocessUrl = (url: string) => {
-        return url;
+        let processedURL = url.substring(5).toLowerCase();
+        if (processedURL.indexOf('./') === 0) {
+            processedURL = processedURL.substring(2);
+          }

```

Comment:
Careful with spacing:
```suggestion
        }
```